provisioning/miniprov/hare_mp/utils.py

```diff
@@ -98,28 +98,44 @@ def _save_drive_info(self, path: str):
         disk_key = path.strip('/')
         self.kv.kv_put(f'{hostname}/{disk_key}', drive_info)
 
+    def is_motr_component(self, machine_id: str) -> bool:
+        """
+        Returns True if motr component is present in the components list
+        for the given node>{machine_id} according to the
+        ConfStore (ValueProvider).
+        """
+        comp_names = self.provider.get(f'node>{machine_id}>'
+                                       f'components')
+        for component in comp_names:
+            if(component.get('name') == Const.COMPONENT_MOTR.value):
+                rc = True
```

Comment:
Why not like this
```suggestion
                return True
```
?

Reply:
This is just by considering the generic rule that there should not be multiple return statements in any function. Then code becomes difficult to understand. Though this function is too small, but this is done for sticking to rule.